apps/lead/models.py

```diff
@@ -277,6 +277,15 @@ def get_authors_display(self):
             return self.author and self.author.data.title
         return self.author_raw
 
+    def get_authoring_organizations_type_display(self):
+        if self.authors.exists():
+            return ','.join(set([
+                author.get_organization_type_display() for author in self.authors.all()
+            ]))
```

Comment:
Let's do it like this. Since we are prefetching, doing `exists` will send another request to the database.
Also, update the `get_authors_display`.

```suggestion
        authors = self.authors.all()
        if authors:
            return ','.join(set([
                author.get_organization_type_display() for author in authors
            ]))
```